src/wbase/SendChannelShared.cc

```diff
@@ -148,23 +150,27 @@ bool SendChannelShared::_addTransmit(bool cancelled, bool erred, bool last, bool
     }
 
     if (reallyLast || erred || cancelled) {
+        LOGS(_log, LOG_LVL_WARN, "&&& SendChannelShared::_addTransmit d _lastRecvd set to true"  << " qId=" << qId << " jId=" << jId);
         _lastRecvd = true;
         LOGS(_log, LOG_LVL_DEBUG, "addTransmit lastRecvd=" << _lastRecvd << " really=" << reallyLast
                                   << " erred=" << erred << " cancelled=" << cancelled);
     }
 
     // If this is reallyLast or at least 2 items are in the queue, the transmit can happen
-    if (_lastRecvd || _transmitQueue.size() >= 2) {
-        bool sendNow = tData->getScanInteractive();
-        // If there was an error, give this high priority.
-        if (erred || cancelled) sendNow = true;
-        int czarId = tData->getCzarId();
-        return _transmit(erred, sendNow, largeResult, czarId);
+    //&&&if (_lastRecvd || _transmitQueue.size() >= 2) {
```

Comment:
is there a point for keeping the commented code here? Why was it wrong? The comment on the previous line is still mentioning "at least 2 items".